src/tests/account/starknet/test_dual_account.cairo

```diff
@@ -51,30 +54,35 @@ fn setup_account_panic() -> (DualCaseAccount, DualCaseAccount) {
 // snake_case target
 //
 
+const NEW_PUBKEY: felt252 = PUBKEY_2;
```

Comment:
I tried to keep the logic of the test case unchanged:

1. setup
2. set NEW_PUB_KEY as account's public key
3. check signature signed with NEW_PUB_KEY

For this flow we can't use `[data.r, data.s]` signature to set new public_key, because we need the special accept_ownership signature for that.

Now, looking into the logic it doesn't make much sense why we have to set a new public key prior to validating the signature. Perhaps, there was some idea behind it at the time of writing the test

We can either keep the logic as it is or get rid of the step of setting new public key and validate a signature signed with the key the account was deployed with. I vote for the latter, if there's no hidden sense in changing the public key

Reply:
Removed this constant, decided to import only Key Pair constants and to access public/private keys as their properties